src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/model/licence/AdditionalCondition.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.publicApi.model.licence
+
+import io.swagger.v3.oas.annotations.media.ArraySchema
+import io.swagger.v3.oas.annotations.media.Schema
+
+@Schema(description = "Describes each additional condition on the licence")
+data class AdditionalCondition(
+
+  @Schema(description = "The additional condition's unique code", example = "5a105297-dce1-4d18-b9ea-4195b46b7594")
+  val code: String,
+
+  @Schema(
+    description = "The category to which the additional condition belongs",
+    example = "Residence at a specific place",
+  )
+  val category: String,
+
+  @ArraySchema(
+    arraySchema = Schema(discriminatorProperty = "type"),
+    schema = Schema(
+      description = "Individual instances of this condition that appear on the licence. Most conditions can only appear on a licence once so would have a single instance but a subset can appear multiple times.",
+      implementation = AdditionalConditionInstance::class,
+    ),
+  )
+  val instances: List<AdditionalConditionInstance>,
+
+)
```

Comment:
two different types of instances using a discriminator so we can only add enumerate the list of electronic monitoring types to specific types of instances - the swagger ui does display this very well due to a very long running bug. 
It produces the correct schema and I've tweaked things so it looks like the standard condition instance type.

Not sure whether instances are making this unnecessary complex.. its slightly odd to the client that they can have a list of instances which will be either standard instances or electronic monitoring instances.

If we just had a flat list of (possibly duplicated) conditions we could have a union of types at the condition level- have a `electronic monitoring condition` type and `standard condition type`..
 
Would end up with some duplicated conditions for MEZ, etc.. though